services/jenkins/jenkins-plugin-installs.service.js

```diff
@@ -23,22 +22,6 @@ const schemaInstallationsPerVersion = Joi.object()
   .required()
 
 export default class JenkinsPluginInstalls extends BaseJsonService {
-  static _getSchema(version) {
-    if (version) {
-      return schemaInstallationsPerVersion
-    } else {
-      return schemaInstallations
-    }
-  }
-
-  static _getLabel(version) {
-    if (version) {
-      return `installs@${version}`
-    } else {
-      return 'installs'
-    }
-  }
```

Comment:
Trying to keep the diff focused on the refactoring to utilize the new render helper, but felt inclined to update this one with our more canonical fetch/transform/render separation of concerns and applied some broader refactoring to that end